core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -2324,12 +2324,18 @@ class KafkaController(val config: KafkaConfig,
 
         case Some(topicName) =>
           topicReq.partitions.forEach { partitionReq =>
+            var isr = List[Int]()
```

Comment:
nit: Using a `var` is not required here. How about using `val isr = if (....) {`?